boards/x86/qemu_x86/qemu_x86_nommu.yaml

```diff
@@ -0,0 +1,9 @@
+identifier: qemu_x86_nommu
+name: QEMU Emulation for X86 (MMU disabled)
+type: qemu
+arch: x86
+toolchain:
+  - zephyr
+supported:
+  - pci
+  - netif:slip
```

Comment:
if you enable this as default then we will have full coverage of userspace/non-userspace for the tests, it will increase the normal sanitycheck run from 1077 to 1237 but this acceptable for making sure we do not break any APIs for non-userspace usage